ocfweb/stats/templates/stats/summary.html

```diff
@@ -48,25 +48,38 @@ <h4>{{printer}}</h4>
         </div>
 
         <h3>Today's Lab Utilization</h3>
-        <table class="table table-striped table-hover">
-            <tr>
-                <th>Desktop</th>
-                <th>Idle (min)</th>
-                <th>Busy (min)</th>
-                <th>% Busy</th>
-            </tr>
-            {% for profile in desktop_profiles %}
+        <table id="todays-lab-utilization" class="table table-striped table-hover">
+            <thead>
                 <tr>
-                    <td>{{profile.hostname}}</td>
-                    <td>{{profile.minutes_idle|floatformat:"0"}}</td>
-                    <td>{{profile.minutes_busy|floatformat:"0"}}</td>
-                    {% if profile.total_minutes <= 0 %}
-                        <td>0%</td>
-                    {% else %}
-                        <td>{{profile.minutes_busy|div:profile.total_minutes|mul:100|floatformat:"0"}}%</td>
-                    {% endif %}
+                    <th>Desktop</th>
+                    <th>Idle (min)</th>
+                    <th>Busy (min)</th>
+                    <th>% Busy</th>
                 </tr>
-            {% endfor %}
+            </thead>
+            <tbody>
+                {% for profile in desktop_profiles %}
+                    <tr>
+                        <td>{{profile.hostname}}</td>
+                        <td>{{profile.minutes_idle|floatformat:"0"}}</td>
+                        <td>{{profile.minutes_busy|floatformat:"0"}}</td>
+                        {% if profile.total_minutes <= 0 %}
+                            <td>0%</td>
+                        {% else %}
+                            <td>{{profile.minutes_busy|div:profile.total_minutes|mul:100|floatformat:"0"}}%</td>
+                        {% endif %}
+                    </tr>
+                {% endfor %}
+            </tbody>
         </table>
     </div>
 {% endblock %}
+
+{% block inline_js %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.tablesorter/2.31.3/js/jquery.tablesorter.min.js"></script>
+<script>
+$(document).ready(function() {
```

Comment:
This line can be replace with just `$(function() {`

Reply:
Shortened it. They are the same though. Plz review :3